src/firefly_exchange_client/api_service.py

```diff
@@ -11,6 +11,12 @@ def __init__(self, url, UUID=""):
         self.uuid = UUID
         self.client = aiohttp.ClientSession()
 
+        
+    def set_uuid(self,uuid):
+        if uuid is not "":
+                self.uuid = uuid
```

Comment:
Not sure if it'll cause an issue, but the indentation here seems to be wrong

Reply:
Just thought of this when reviewing the TS client:

> Why are we checking that the string isn't ""? In the case that someone wants to unset their UUID, they wouldn't be able to. Maybe we just skip that check?

Reply:
I just was not sure if an empty UUID is considered as a unique UUID and rate limits are assigned uniquely to it. So wanted to remove the header overall if the UUID field is not present. But you are right about atleast letting the user set their UUID to empty if they no longer wish to use it.